### ISSUE TITLE
No emoji output to terminal

### ISSUE THREAD

**Issue by mjKosmic** [GaZaTu/x11-emoji-picker#21]
I am running awesomewm with alacritty and i can output emojies just fine to GUI applications like a web browser or discord, but alacritty and vim/neovim just don't like something and refuse to receive the emojis correctly

**Comment by GaZaTu** [comment]
Sorry but i can't reproduce this issue.
When you say "correctly" what does that exactly mean?
And which input method are you using?

![Screenshot_20220312_093551](https://user-images.githubusercontent.com/17497826/158010663-8f4c9917-9570-4d8b-9440-80b996dace35.png)

**Comment by mjKosmic** [comment]
It's with any of the available methods. It looks like it's not setting my copy-paste buffer. When i select an emoji, it transfers focus back to the original window just fine but doesn't output anything. If i copy some other text, then try to select an emoji, i can still paste the other text i copied previously

**Comment by mjKosmic** [comment]
Interestingly if i press enter a couple of times this shows up
![screenshot](https://user-images.githubusercontent.com/8421227/158027688-90574ad6-8bb2-41c4-95ab-c4f621200aa0.png)

**Comment by GaZaTu** [comment]
It shouldn't even need copy-paste.
Maybe i have to setup a vm with awesomewm to test it.

**Comment by GaZaTu** [comment]
I just now had pretty much the same issue: emojis wouldn't enter in thunderbird or konsole.
But i found that copy pasting using ctrl+a & ctrl+c into an editor where it did work (in your case firefox i guess), did reveal that emojis were written and just not visible in thunderbird because of the selected emoji font in my fontconfig. After i changed it to for example "Noto Emoji", it worked again.

So what I'm saying is: try to check if emojis **really** aren't written or if they're just invisible. You can do this by copying an emoji from here maybe ( 😋 ) and pasting it into alactritty.

**Comment by eclairevoyant** [comment]
First line is me cycling through all 4 input methods
Second line is me pasting emoji from a GUI application directly into the terminal

I also think ctrl+V is not the best way to handle pasting on linux, but maybe that mode was intended for windows

![2022-07-24_12-56](https://user-images.githubusercontent.com/848000/180658004-7edbc27b-d9ef-4f0e-b809-034f98313828.png)

**Comment by GaZaTu** [comment]
You're right, using ctrl+v to paste into a terminal shouldn't normally work. 😅
But as i said above, the default input method should work just fine with alacritty.
I wonder if this is simply some kind of timing issue where the clipboard isn't updated in time for the ctrl+v request because i also get similar issues when trying to insert alot of emojis in a row.
Now i could add some kind of sleep into the insert function and wait idk 100ms but idk. 😶
